Dragonflight/AlgetharAcademy/Locales/zhCN.lua

```diff
@@ -2,22 +2,22 @@ local L = BigWigs:NewBossLocale("Algeth'ar Academy Trash", "zhCN")
 if not L then return end
 if L then
 	L.recruiter_autotalk = "自动对话"
-	--L.recruiter_autotalk_desc = "Instantly pledge to the Dragonflight Recruiters for a buff."
+	L.recruiter_autotalk_desc = "立即找巨龙招募人员获取增益"
 	L.critical_strike = "+5% 爆击"
 	L.haste = "+5% 急速"
-	L.mastery = "+精通"
+	L.mastery = "+97精通"
```

Comment:
We can't include the number in the message, 97, because it scales based on level or difficulty.